istioctl/cmd/auth.go

```diff
@@ -273,19 +284,12 @@ func init() {
 		"Check the TLS/JWT/RBAC setting from the config dump file")
 	validatorCmd.PersistentFlags().StringSliceVarP(&policyFiles, "file", "f", []string{},
 		"Authorization policy file")
-	upgradeCmd.PersistentFlags().StringSliceVarP(&policyFiles, "file", "f", []string{},
-		"Authorization policy files")
-	upgradeCmd.PersistentFlags().StringSliceVarP(&serviceFiles, "service", "s", []string{},
-		"Kubernetes Service resource that provides the mapping relationship between service name and pod labels")
-	upgradeCmd.PersistentFlags().StringVarP(&meshConfig, "meshConfigFile", "m", "",
-		"Istio MeshConfig file")
-	upgradeCmd.PersistentFlags().StringVar(&istioMeshConfigMapName, "meshConfigMapName", "istio",
+	convertCmd.PersistentFlags().StringSliceVarP(&policyFiles, "file", "f", []string{},
+		"v1alpha1 RBAC policy that needs to be converted to v1beta1 authorization policy")
+	convertCmd.PersistentFlags().StringSliceVarP(&serviceFiles, "service", "s", []string{},
+		"Kubernetes Service resource that provides the mapping between service and workload")
+	convertCmd.PersistentFlags().StringVarP(&meshConfig, "meshConfigFile", "m", "",
+		"Istio MeshConfig file that provides the root namespace value")
+	convertCmd.PersistentFlags().StringVar(&istioMeshConfigMapName, "meshConfigMapName", "istio",
 		"ConfigMap name for Istio mesh configuration")
 }
-
-// cleanupForTest clean the values of policyFiles and serviceFiles. Otherwise, the variables will be
-// appended with new values
-func cleanupForTest() {
```

Comment:
Why deleted this? This is needed for test (unit tests for the tool are failing), since the test loads the same package-scoped variables and we're most likely modifying those variable for each test run in the conversion tests. I think we can solve this by calling this function in the test instead.

Reply:
I'm not very clear why do we need this but I'm sure we shouldn't put in the normal path. I moved this to `auth_test.go` in TestAuthConvert.

Reply:
Yes, making the function public so we can clear the variables in the tests will be better. Thanks! We need it because many of the variables we're using in the auth.go files are package-scoped variables.